### ISSUE TITLE
How can I elaborate 500 images?

### ISSUE THREAD

**Issue by Cioscos** [vt-vl-lab/FGVC#36]
Hi! I've problems to elaborate 500 images because I go in out of memory of ram. Moreover I tried with 2 images and It went in OOM of VRAM. Maybe resolution must be lower?

**Comment by gaochen315** [comment]
Yes, it depends on the resolution as well because we need to feed the whole video to the memory. What is the resolution?

**Comment by Cioscos** [comment]
Resolution is full hd. Does it works with videos or just images?

**Comment by aklos** [comment]
From what I've tried, the max res is ~1280x768 with 24GB of VRAM.

**Comment by xenjee** [comment]
FWIW, just did a test on 1920x1080 14 frames and it went up to about 22.8GB for this process.
I had another app running at the same time taking an extra 5GB, so that passed 24GB. Running on an RTX8000 (48GB)

**Comment by jx98398534** [comment]
good
